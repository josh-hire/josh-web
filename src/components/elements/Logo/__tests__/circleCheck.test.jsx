import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { IconCircleCheck } from '../circleCheck';

describe('src/components/elements/circleCheck', ()=>{
  test('circleCheck',()=>{
    const { asFragment } = render(<IconCircleCheck />);
    expect(asFragment()).toMatchSnapshot();
  });
});
