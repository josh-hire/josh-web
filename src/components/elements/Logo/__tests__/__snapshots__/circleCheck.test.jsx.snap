// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/circleCheck circleCheck 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="48"
    viewBox="0 0 48 48"
    width="48"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M24 4C35.0457 4 44 12.9543 44 24C44 35.0457 35.0457 44 24 44C12.9543 44 4 35.0457 4 24C4 12.9543 12.9543 4 24 4ZM30.4393 17.9393L21.5 26.8787L17.5607 22.9393C16.9749 22.3536 16.0251 22.3536 15.4393 22.9393C14.8536 23.5251 14.8536 24.4749 15.4393 25.0607L20.4393 30.0607C21.0251 30.6464 21.9749 30.6464 22.5607 30.0607L32.5607 20.0607C33.1464 19.4749 33.1464 18.5251 32.5607 17.9393C31.9749 17.3536 31.0251 17.3536 30.4393 17.9393Z"
      fill="#44AB49"
    />
  </svg>
</DocumentFragment>
`;
